docs/api/model.md

```diff
@@ -206,30 +221,32 @@ The success listener is called with an array of instances if the query succeeds.
 | [options.include[].where] | Object | Where clauses to apply to the child models. Note that this converts the eager load to an inner join, unless you explicitly set `required: false` |
-| [options.include[].attributes] | Array&lt;String&gt; | A list of attributes to select from the child model |
+| [options.include[].attributes] | Array.&lt;String&gt; | A list of attributes to select from the child model |
 | [options.include[].required] | Boolean | If true, converts to an inner join, which means that the parent model will only be loaded if it has any matching children. True if `include.where` is set, false otherwise. |
-| [options.include[].include] | Array&lt;Object &#124; Model&gt; | Load further nested related models |
-| [options.order] | String &#124; Array &#124; Sequelize.fn | Specifies an ordering. If a string is provided, it will be esacped. Using an array, you can provide several columns / functions to order by. Each element can be further wrapped in a two-element array. The first element is the column / function to order by, the second is the direction. For example: `order: [['name', 'DESC']]`. In this way the column will be escaped, but the direction will not. |
+| [options.include[].through.where] | Object | Filter on the join model for belongsToMany relations |
+| [options.include[].through.attributes] | Array | A list of attributes to select from the join model for belongsToMany relations |
+| [options.include[].include] | Array.&lt;Object &#124; Model&gt; | Load further nested related models |
+| [options.order] | String &#124; Array &#124; Sequelize.fn | Specifies an ordering. If a string is provided, it will be escaped. Using an array, you can provide several columns / functions to order by. Each element can be further wrapped in a two-element array. The first element is the column / function to order by, the second is the direction. For example: `order: [['name', 'DESC']]`. In this way the column will be escaped, but the direction will not. |
 | [options.limit] | Number |  |
 | [options.offset] | Number |  |
 | [options.transaction] | Transaction |  |
-| [queryOptions] | Object | Set the query options, e.g. raw, specifying that you want raw data instead of built Instances. See sequelize.query for options |
-| [queryOptions.lock] | String|Object | Lock the selected rows. Possible options are transaction.LOCK.UPDATE and transaction.LOCK.SHARE. Postgres also supports transaction.LOCK.KEY_SHARE, transaction.LOCK.NO_KEY_UPDATE and specific model locks with joins. See [transaction.LOCK for an example](api/transaction#lock)  |
+| [options.lock] | String | Lock the selected rows in either share or update mode. Possible options are transaction.LOCK.UPDATE and transaction.LOCK.SHARE. See [transaction.LOCK for an example](https://github.com/sequelize/sequelize/wiki/API-Reference-Transaction#LOCK) |
```

Comment:
True, I made a mistake changing the description in the wrong place. I'll fix that later / tomorrow.